superset/connectors/sqla/models.py

```diff
@@ -312,6 +312,10 @@ def explore_url(self) -> str:
     def column_formats(self) -> dict[str, str | None]:
         return {m.metric_name: m.d3format for m in self.metrics if m.d3format}
 
+    @property
+    def currency_formats(self) -> dict[str, dict[str, str | None] | None]:
+        return {m.metric_name: m.currency for m in self.metrics if m.currency}
+
```

Comment:
This is just sending the same thing that we're already sending. This data structure should be built by the frontend if it needs the currencies in a single dictionary.